### PR TITLE
GG-28983 Agent ignores management-truststore and management-truststore-password parameters

### PR DESCRIPTION
**Pull request by nva**

### CODE REVIEW

File: modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java [hunk 1]
```diff
@@ -161,15 +161,15 @@ public ManagementConfiguration setConsoleTrustStore(String consoleTrustStore) {
      * @return Control Center trust store password.
      */
     public String getConsoleTrustStorePassword() {
```
[the remark this answers is not in the document]
Comment (nva) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java]: You are wright. Fixed.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java [hunk 2]
```diff
@@ -161,15 +161,15 @@ public ManagementConfiguration setConsoleTrustStore(String consoleTrustStore) {
...
-        return consoleTrustStorePass;
+        return trustStorePass;
     }
 
     /**
      * @param consoleTrustStorePass Console trust store password.
      * @return {@code this} for chaining.
      */
     public ManagementConfiguration setConsoleTrustStorePassword(String consoleTrustStorePass) {
```
[the remark this answers is not in the document]
Comment (nva) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java]: You are wright. Fixed.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java [hunk 3]
```diff
@@ -127,32 +127,32 @@ public ManagementConfiguration setConsoleKeyStore(String consoleKeyStore) {
      * @return Control Center key store password.
      */
     public String getConsoleKeyStorePassword() {
```
[the remark this answers is not in the document]
Comment (nva) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/management/ManagementConfiguration.java]: You are wright. Fixed.

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/management/ManagementCommands.java [hunk 4]
```diff
@@ -151,11 +158,11 @@ private void printHelp(Logger log) {
 
                 while (argIter.hasNextSubArg()) {
                     ManagementURLCommandArg uriArg = CommandArgUtils.of(
-                        argIter.nextArg("Expected one of uri arguments"), ManagementURLCommandArg.class
+                        argIter.nextArg("Invalid uri arguments"), ManagementURLCommandArg.class
                     );
 
                     if (uriArg == null)
-                        throw new IllegalArgumentException("Expected one of auto-adjust arguments");
+                        throw new IllegalArgumentException("Invalid uri arguments");
```
Comment (MaxLosevskoy) [inline modules/core/src/main/java/org/apache/ignite/internal/commandline/management/ManagementCommands.java:165]: Can we extend the message to highlight what exact arg is _null_?

File: modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java [hunk 5]
```diff
@@ -690,4 +694,67 @@ private boolean requireArgs(@Nullable CommandList cmd) {
...
+        assertFalse(arg.isEnable());
+
+        assertParseArgsThrows("Expected server URIs", "--management", "uri");
+
+        args = parseArgs(asList("--management", "uri", "http://localhost:3000"));
+
+        arg = ((ManagementCommands)args.command()).arg();
+        
+        assertTrue(arg.isEnable());
+        assertEquals(singletonList("http://localhost:3000"), arg.getServerUris());
```
Comment (MaxLosevskoy) [inline modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java:727]: Is that fine to hardcode the port?

File: modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java [hunk 6]
```diff
@@ -690,4 +694,67 @@ private boolean requireArgs(@Nullable CommandList cmd) {
...
+
+        assertParseArgsThrows("Invalid uri arguments", "--management", "uri", "http://localhost", "--wrong");
+
+        String keyStorePath = U.resolveIgnitePath("/modules/core/src/test/resources/server.jks").getAbsolutePath();
```
Comment (MaxLosevskoy) [inline modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java:731]: Alternatively, from com.google.common.io.Resources
```java
getResource("server.jks");
```

File: modules/core/src/main/java/org/apache/ignite/internal/commandline/management/ManagementCommands.java (hunk 4 above)
Comment (nva) [reply modules/core/src/main/java/org/apache/ignite/internal/commandline/management/ManagementCommands.java]: Done

File: modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java (hunk 5 above)
Comment (nva) [reply modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java:727]: Yes

File: modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java (hunk 6 above)
Comment (nva) [reply modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java:731]: I don't want add dependency from guava

Comment (MaxLosevskoy) [reply modules/core/src/test/java/org/apache/ignite/internal/commandline/CommandHandlerParsingTest.java:731]: It is there since GG-27090

**Pull request closed without merging**